pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -456,3 +461,34 @@ func fixLibvirtCrcNetworkActive() (bool, error) {
 	}
 	return true, nil
 }
+
+func checkCrcDnsmasqConfigFile() (bool, error) {
+	c := []byte(crcDnsmasqConfig)
+	_, err := os.Stat(crcDnsmasqConfigPath)
+	if err != nil {
+		return false, fmt.Errorf("File not found: %s: %s", crcDnsmasqConfigPath, err.Error())
+	}
+	config, err := ioutil.ReadFile(crcDnsmasqConfigPath)
+	if err != nil {
+		return false, fmt.Errorf("Error opening file: %s: %s", crcDnsmasqConfigPath, err.Error())
+	}
+	if len(config) != len(c) {
+		return false, fmt.Errorf("File seems corrupt: %s: %s", crcDnsmasqConfigPath)
+	}
+	if !bytes.Equal(config, c) {
```

Comment:
If we are already comparing the bytes the why we need `len` also?

Reply:
same thought here...